security/pfSense-pkg-tinc/files/usr/local/www/status_tinc.php

```diff
@@ -26,8 +26,15 @@ function tinc_status_usr1() {
 	usleep(500000);
 	$clog_path = "/usr/local/sbin/clog";
 	$result = array();
+	$logfile = "/var/log/tinc.log";
+	$firmware = host_firmware_version();
 
-	exec("{$clog_path} /var/log/tinc.log | /usr/bin/sed -e 's/.*tinc\[.*\]: //'", $result);
+	if ($firmware['config_version'] >= 19.7) {
+		exec(system_log_get_cat() . ' ' . $logfile . "| /usr/bin/sed -e 's/.*tinc\[.*\]: //'", $result);
+	} else {
+		exec("{$clog_path} /var/log/tinc.log | /usr/bin/sed -e 's/.*tinc\[.*\]: //'", $result);
```

Comment:
Why use the full hardcoded path to the log here, rather than `{$logfile}` which is populated above?